colossus-tests/src/test/scala/colossus/core/ConnectionHandlerSpec.scala

```diff
@@ -92,7 +92,7 @@ class ConnectionHandlerSpec extends ColossusSpec {
       }
       withIOSystem{ implicit io =>
         //obvious this will fail to connect, but we don't care here
-        io ! IOCommand.BindAndConnectWorkerItem(new InetSocketAddress("localhost", TEST_PORT), new MyHandler)
+        io ! IOCommand.BindAndConnectWorkerItem(new InetSocketAddress("localhost", TEST_PORT), _ => new MyHandler)
```

Comment:
Bit of a drive by comment, as I'm about to run out the door, and Naomi is on the tear, but, here is a slightly half baked thought:

object BindAndConnectWorkerItem {
    implicit def liftItemToFunc(item : WorkerItem) : (WorkerRef => WorkerItem) = _ => item
  }

Then, in these places where we are really just using the function to wrap an instance, we can just import that implicit and the code doesn't change here, it will remain 
io ! IOCommand.BindAndConnectWorkerItem(new InetSocketAddress("localhost", TEST_PORT), new MyHandler)

Thoughts?